docs/installation.md

```diff
@@ -2,20 +2,31 @@
 
 ## Installation
 
-First, make sure you have [created and activated](https://docs.python.org/3/library/venv.html) a Python3.8+ virtual environment.
+First, ensure you have [created and activated](https://docs.python.org/3/library/venv.html) a Python 3.8+ virtual environment.
```

Comment:
The link for creating and activating a virtual env is for python venv; however our first suggested option is then to use uv. I am not sure about interoperability between the two.

I am a big fan of mkdocs material [content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks) to have both option displayed and aligned when selecting one or the other.

What do you think?

Reply:
Good iead!